src/Sylius/Bundle/CoreBundle/Controller/ProductTaxonController.php

```diff
@@ -99,13 +145,13 @@ private function shouldProductsPositionsBeUpdated(Request $request, ?array $prod
         return in_array($request->getMethod(), ['POST', 'PUT', 'PATCH'], true) && null !== $productTaxons;
     }
 
-    private function updatePositions(string $position, int $id): void
+    private function updatePositions(int $position, int $id): void
```

Comment:
This will result in a 500 in `::updatePositionsAction` as the validation is missing.